includes/class-poet-backfill.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Feature to backfill old posts to Frost API.
+ *
+ * @TODO Make a WP-CLI for this as well so one could batch process tens/hundreds of thousands of posts
+ *
+ * @package    Poet
+ * @subpackage Poet/includes
+ */
+class Poet_Backfill {
+
+    /**
+     * Schedule a cron to backfill posts if the user has this feature enabled.
+     */
+    public static function init() {
+        $should_backfill = isset( get_option( 'poet_option' )['active'] ) ? 1 : 0;
```

Comment:
@markparolisi shouldn't this be `$should_backfill = isset( get_option( 'poet_option' )['backfill'] ) ? 1 : 0;` rather than `active`?

Reply:
I still can't get the backfill to work though, even when making that change. I'm not familiar with these functions though

Reply:
Me neither. I enabled the option but the previous post that was not set with a badge still does not show one.

Reply:
yes it should. 
Relatedly, I haven't tested this because I need to setup a token in the API. I referenced this with my "cowboy coding" in the email with Geoff. I'm sure there are one or more small errors like this I need to patch, but I need some time to sit down and config an end to end test env. Sorry I've been too busy and just slinging code out.

Reply:
I've figured out a few oddities based on the way the post meta for `poet_work_id` is stored. This is causing problems with the database query, specifically around this line, https://github.com/poetapp/wordpress-plugin/blob/master/public/class-poet-public.php#L140
Don't think that should be there. It prevents one from doing a `NOT EXISTS` query. There are workarounds though.